skfp/model_selection/randomized_scaffold_split.py

```diff
@@ -369,13 +355,17 @@ def _create_scaffolds(
     molecules = ensure_mols(data)
 
     for idx, mol in enumerate(molecules):
+        mol = deepcopy(mol)
+        Chem.RemoveStereochemistry(mol)  # important for canonization of CSK!
+        scaff = MurckoScaffold.GetScaffoldForMol(mol)
+        scaff = MurckoScaffold.MakeScaffoldGeneric(scaff)
+        scaff = MurckoScaffold.GetScaffoldForMol(scaff)
```

Comment:
Too much copy-paste, you always should do `RemoveSterochemistry`, rest is only for CSK